sbt-scalajs-bundler/src/main/scala/scalajsbundler/PackageJson.scala

```diff
@@ -42,7 +42,8 @@ object PackageJson {
       ) ++ Seq(
         "webpack" -> webpackVersion,
         "concat-with-sourcemaps" -> "1.0.4", // Used by the reload workflow
-        "source-map-loader" -> "0.1.5" // Used by webpack when emitSourceMaps is enabled
+        "source-map-loader" -> "0.1.5", // Used by webpack when emitSourceMaps is enabled
+        "webpack-dev-server" -> "1.16.2"
```

Comment:
I would rather install it independently, like we do for jsdom, instead of adding it to the project’s dependencies.